lib/SILOptimizer/Transforms/SemanticARCOpts.cpp

```diff
@@ -52,21 +52,36 @@ class LiveRange {
   /// introducer and not to be forwarding.
```

Comment:
MG: Rename to OwnershipLiveRange and add a comment that says that this is an extended live range that looks through forwarding instructions but not phis. Explain how this is a building block for the phi web.